episodes/16-what-is-myst.Rmd

```diff
@@ -142,6 +174,43 @@ We find that the the file holds some metadata for the webpage. Most of it isn't
 So we learn that myst requires this file to be in your project directory to be able to built the webpage.
 
 
+::::::::::::::::::::::::::::::::::::: challenge 
```

Comment:
before the challenge, let's plan to edit something together in the instructional part and then leave the challeng to edit another 

so maybe we edit the authors first? and leave title as the exercise or make title the instruction and edit the author as exercise?